dpdd.tex

```diff
@@ -93,6 +93,7 @@
 \addtohist{3.0}{2017-07-03}{Implementation of LCR-962. Reference LSE-61 requirements. Minor clean ups.}{Tim Jenness}
 \addtohist{3.1}{2018-01-04}{Implementation of LCR-1024. Mention relevant OSS requirements.}{Tim Jenness}
 \addtohist{3.2}{2018-02-09}{Implementation of LCR-1214. Update Level 1, Level 2 and Level 3 nomenclature as defined in \citeds{LPM-231}. Shrink section Level 3 section and refer to Science Platform.}{R.~Gill, W.~O'Mullane}
+\addtohist{3.3}{2018-02-28}{Implemented adoption of nanojansky flux unit, added point estimates for photo-z, and a few minor edits.}{\v{Z}.~Ivezi\'{c}}
```

Comment:
The version should be set when the document is baselined, not when this is merged to master. Adding the changelog entry is nice, but I'd have version number “TBD” or similar.